config/base/kustomization.yaml

```diff
@@ -12,4 +12,6 @@ resources:
 - kong-ingress-dbless.yaml
 
 components:
-  - ../image/oss
+- ../image/oss
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
```

Comment:
This is a good change to explicitly add those fields yet, conventionally, we're trying to keep those as the first lines in a file, e.g. https://github.com/Kong/kubernetes-ingress-controller/blob/6a0b79add0a89c331ccdaca94778e4ce4fb6c3d0/config/variants/multi-gw/base/kustomization.yaml#L1-L2